docs/ble.md

```diff
@@ -92,6 +89,48 @@ interface Characteristic {
 
 API Documentation
 -----------------
+BLE is an [EventEmitter](./events.md) with the following events:
+
+### Event: 'accept'
+
+* `string` `clientAddress`
+
+Emitted when a BLE client has connected. *NOTE*: currently the address is the
+dummy string "AB:CD:DF:AB:CD:EF".
+
+### Event: 'advertisingStart'
+
+* `int` `status`
+
+Emitted when BLE services have begun to be advertised. The `status` will be 0
+for success, otherwise for an error.
+
+### Event: 'disconnect'
+
+* `string` `clientAddress`
+
+Emitted when a BLE client has disconnected. *NOTE*: currently the address is the
+dummy string "AB:CD:DF:AB:CD:EF".
+
+### Event: 'rssiUpdate'
+
+* `int` `rssiValue`
+
+Emitted periodically with most recent RSSI (received signal strength indication)
+value measured in dBm. *NOTE*: currently rssiValue is the dummy value -50.
+
+FIXME: @jimmy-huang, while connect/disconnect are still sort of useful with a
```

Comment:
real address it not possible until a client is initiating a pairing process, I think this would be something we can ask for Zephyr Bluetooth team to see if they can add this.